src/Smalot/PdfParser/RawData/RawDataParser.php

```diff
@@ -888,7 +882,7 @@ public function parseData(string $data): array
         }
 
         // get PDF content string
-        $pdfData = substr($data, $trimpos);
+        $pdfData = $trimpos > 0 ? substr($data, $trimpos) : $data;
 
```

Comment:
Be reasonably lazy: in most cases $trimpos would be zero. If you should do nothing -- do nothing.

Probably, It would be better to use $offset parameter of the getXrefData($pdfData); instead of substring. But I'm afraid of this comment:
* @param int   $offset xref offset (if known)
and strange if in the protected function getXrefData
```php
if (0 == $offset) {
            // find last startxref
```
Seems, zero offset is really a special case.